### PR TITLE
Add lease renewal time

### LINKED ISSUE #14815: Recovery race
Peers can detect each other's deaths prematurely under certain circumstances.

### PR DESCRIPTION
**Pull request by jonhawkes**
fixes #14815 
#build

### CODE REVIEW

File: dev/com.ibm.ws.transaction/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -87,3 +87,6 @@ lease.check.interval.desc=The interval after which the lease check strategy is i
 
 lease.length=Lease length
 lease.length.desc=The length of time before a recovery log lease expires
+
+lease.renewal.time=Lease Renewal Time
+lease.renewal.time.desc=The percentage of the lease length after which the lease is renewed
```
Comment (dmuelle) [inline dev/com.ibm.ws.transaction/resources/OSGI-INF/l10n/metatype.properties:92]: Suggested edit:

`The percentage of the duration of the recovery log lease that passes before the lease is renewed`

**Review by dmuelle — CHANGES_REQUESTED** [review]

**Comment by jonhawkes** [comment]
#build 
#run-libby-bot

**Review by dmuelle — APPROVED** [review]
LGTM

**Comment by andrea-pichler** [comment]
No message review requested from L2

**Comment by jonhawkes** [comment]
#build 
#run-libby-bot

**Pull request merged**